data/output/presale.rinkeby.json

```diff
@@ -24,7 +24,11 @@
     "transactionHash": "0x00ed03f2dcdae1acfdf12e183ee5521cfa4dadedf4f53d4af8569da3e9340f23"
   },
   "wrapper": {
+    "address": "0x86dfd24835732c1283b259155aae0f8ffd9e0715",
+    "transactionHash": "0xc48d49f8b704aa6db1805983717a94bdf8f00a8e741d9a939fdca71dda314fc9"
+  },
+  "wrapper-deprecated": {
```

Comment:
Did you add this manually? These files are auto-generated, and in order to re-deploy must be deleted.

Reply:
Actually for wrapper there's no need to delete it, so it's fine.

Reply:
Actually for wrapper there's no need to delete it, so it's fine.

Reply:
Maybe there's a better way to do this later but yes, I did it manually to mark the old ones as deprecated.